keps/sig-node/1967-size-memory-backed-volumes/README.md

```diff
@@ -141,115 +174,139 @@ potentially inaccurate volume size based on node configuration.
 
 ### Feature Enablement and Rollback
 
-_This section must be completed when targeting alpha to a release._
+#### How can this feature be enabled / disabled in a live cluster?
 
-* **How can this feature be enabled / disabled in a live cluster?**
-  - [x] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: SizeMemoryBackedVolumes
-    - Components depending on the feature gate: kubelet
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? No
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: SizeMemoryBackedVolumes
+  - Components depending on the feature gate: kubelet
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? No
+
+#### Does enabling the feature change any default behavior?
 
-* **Does enabling the feature change any default behavior?**
 Yes, the kubelet will size the empty dir volume to match the precise
 amount of memory the pod is able to write rather than over or undersizing.
 Prior behavior is node dependent, and so pod authors had no mechanism
 to control this behavior properly.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes
+#### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement?
+
+Yes
+
+#### What happens if we reenable the feature if it was previously rolled back?
 
-* **What happens if we reenable the feature if it was previously rolled back?**
 Pods that run on that node will have memory backed volumes sized based on Linux
 host default.  The sizing may not align with actual available memory for an app.
 
-* **Are there any tests for feature enablement/disablement?**
+#### Are there any tests for feature enablement/disablement?
+
 No, testing behavior with the feature disabled is dependent on node operating
```

Comment:
Seems like tests exist verifying just that, see https://github.com/kubernetes/kubernetes/blob/f7fef0384ec1c666e6b68cae83a3b3c6d95aa7b0/pkg/volume/emptydir/empty_dir_test.go#L1046-L1063

Reply:
updated.